Portland 2022 full website and ticket sales blog post (#1661)
writethedocs/www: docs/conf/portland/2022/news/tickets-on-sale.rst

:template: {{year}}/generic.html

> .. post:: Jan 7, 2021

Author: ```suggestion
.. post:: Feb 2, 2022
```

   :tags: portland-2021, tickets

Announcing ticket sales, full conference website, and CFP extension
===================================================================

Greeting, documentarians! Following our announcement that the Portland conference will remain virtual only, we're excited to announce that tickets are on sale and the full website is now live.

Following the success of our previous virtual conferences, we look forward to more opportunities to bring the community together virtually and to share the wonderful energy with each other wherever you are in the world.

Get your ticket for the conference
----------------------------------

**The Portland conference will be held online on {{ date.short }}.** The `full event website <https://www.writethedocs.org/conf/portland/{{year}}/>`_ is live and already includes some preliminary details. We'll be updating it with more information as we get closer to the event. 

> If you're excited too, go ahead and `register for your ticket now <https://www.writethedocs.org/conf/portland/{{year}}/tickets/>`_! We aren't sure exactly what capacity we will be able to hold for the virtual event, but we might still sell out to make sure we're providing a manageable event size for our attendees, as always.

Reviewer: This feels odd given we've had many virtual conferences..

```suggestion
If you're excited too, go ahead and `register for your ticket now <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/tickets/>`_! We are still planning to limit capacity of the event to ensure that we have a good experience for everyone, but normally that happens 1-2 weeks before the conference date.
```

Similar to last year's virtual conference, we're going to keep as much of the spirit of the event as possible:

> * Excellent single track of pre-recorded presentations with moderated Q&A for each talk

Reviewer: A bit more clear about not having individual Q&A

```suggestion
* Excellent single track of pre-recorded presentations with moderated Q&A of speakers
```

* Writing Day
* Unconference sessions
* Job Fair
* Sponsor booths
* A virtual hallway track

> * A social event of some kind, bring your own beverage and snacks :)

Reviewer: ```suggestion
* A social event, but you'll need to bring your own beverage and snacks :)
```

> We really want to make sure that folks can get value from being at the event at the same time, and we think it will be another great year of helping our community connect.

Reviewer: ```suggestion
We plan the event to ensure lots of interaction, which is the main reason we're all gathered together, and we think it will be another great year of helping our community connect.
```

> Submit your talk by **February 25th, 2022**

Reviewer: ```suggestion
Submit your talk by **{{ cfp.ends }}**
```

--------------------------------------------------

Following the decision to keep the conference virtual, we are extending the Call for Proposals to give folks more time to prepare their proposals. Check out the `Call for Proposals page <https://www.writethedocs.org/conf/portland/{{year}}/cfp/>`_ for more details about what we are looking for.

If you already have an idea but you’re having a hard time getting your proposal to come together, you can always ask for feedback on the `Write the Docs Slack <https://www.writethedocs.org/slack/>`_. The community is there to help!

> In addition to the online resources, the Write the Docs East Coast Quorom meetup will hold a `Conference Proposal Workshop on February 10th in US Eastern time zone <https://www.meetup.com/virtual-write-the-docs-east-coast-quorum/events/283502054/>`_. Join your fellow documentarians to discuss potential proposals, brainstorm ideas, and get feedback on your proposal before you submit. Big shout-out to Alyssa Rock and the organizers of this Quorom meetup!

Author: ```suggestion
In addition to the online resources, the Write the Docs East Coast Quorum meetup will hold a `Conference Proposal Workshop on February 10th in US Eastern time zone <https://www.meetup.com/virtual-write-the-docs-east-coast-quorum/events/283502054/>`_. Join your fellow documentarians to discuss potential proposals, brainstorm ideas, and get feedback on your proposal before you submit. Big shout-out to Alyssa Rock and the organizers of this Quorum meetup!
```

After we review all the talk proposals, and will be contacting everyone who submitted a proposal by {{ cfp.notification }}.

Grant program
-------------

> We updated our grant program application to take into account the virtual format of the event.

Reviewer: ```suggestion
We are continuing our grant program again this year in a similar format. 
```

We're excited to be able to offer free tickets to a larger number of people this year, since there are now minimal other costs to attend the event.

You can apply by **{{ grants.ends }}** on `our website <https://www.writethedocs.org/conf/portland/{{year}}/opportunity-grants/>`_.

Support our community by sponsoring
-----------------------------------

Every year we are fortunate to have the support of like-minded organizations and communities, who believe in the collaborative and welcoming community spirit and help us make our events great.

We have published our `virtual sponsorship prospectus`_ for the conference,
and are open for new sponsorship inquiries.

> .. _virtual sponsorship prospectus: https://www.writethedocs.org/conf/portland/2021/sponsors/prospectus/

Author: ```suggestion
.. _virtual sponsorship prospectus: https://www.writethedocs.org/conf/portland/2022/sponsors/prospectus/
```
